### PR TITLE
enh(chore) add typescript node config

### PR DESCRIPTION
**Pull request by karimsx**

### CODE REVIEW

File: eslint/node/typescript.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+module.exports = {
```
Comment (bdauria) [inline eslint/node/typescript.eslintrc.js:1]: Please align the formatting to our actual eslint config

Comment (karimsx) [reply eslint/node/typescript.eslintrc.js:1]: fixed

**Comment by bdauria** [comment]
@karimsx it seems that the config is duplicated from the existing one, can we reuse it and extend it instead?

**Comment by karimsx** [comment]
> instead

yes i will try to refactor typescript global config

**Comment by bdauria** [comment]
@karimsx can you check that centren stil works with this updated configuration before merging?

**Comment by karimsx** [comment]
> @karimsx can you check that centren stillworks with this updated configuration before merging?

just checked and fixed the eslint config, should be working right now

**Review by Thebarda — APPROVED** [review]
Be careful about the breaking changes !

**Review by bdauria — APPROVED** [review]

**Comment by karimsx** [comment]
> Be careful about the breaking changes !

what breaking changes ?

**Comment by Thebarda** [comment]
My bad, I thought you have changed the path to the Typescript ESLint configuration ^^

**Pull request merged**